### ISSUE TITLE
Error 6002 Alternate Animations (XP32)

### ISSUE THREAD

**Issue by theenigmamachine** [ShikyoKira/Project-New-Reign---Nemesis-Main#208]
Hey, I get this error when trying to launch the nemesis behaviour engine: 

ERROR(6002): An exception has occured while patching. Report to Nemesis' author immediately
File: Alternate Animation
Message: boost::filesystem::copy_file: Ingen tilgang: "Papyrus Compiler\backup scripts\Actor.psc", "Papyrus Compiler\scripts\Actor.psc"

I've tried to reinstall Nemesis and XP32 multiple times, restarted my computer, and done what I've read on here and what I thought would work, but to no avail. It seems to work without XP32 installed, but I need it for some other mods. I'm using MO2 btw. 

I'm quite unfamiliar with these things, so I'm going to assume I'm just stupid and it's probably pretty obvious, but all help would be appreciated. :)

**Comment by ShikyoKira** [comment]
May i know whats the meaning of "Ingen tilgang"?

**Comment by theenigmamachine** [comment]
My God I'm sorry, it's "No access"

**Comment by theenigmamachine** [comment]
I finally found out what as causing the issue. In my MO2 "overwrites" the nemesis files "Papyrus Compiler\backup scripts\Actor.psc" were there, but the Actor.psc was just an empty folder, not the actual .psc file, so I copied the real Actor.psc from the same "Papyrus Compiler\backup scripts\Actor.psc" location in my MO2 mods folder, and pasted it in the MO2 "overwrites" where it was supposed to be. I don't know why it created a fake folder, but alas, it worked.
